content/en/docs/concepts/workloads/pods/probes.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Probes"
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+This page provides an overview of Probes. Probes in Kubernetes serve as a health checker for the containers, ensuring their reliability and availability within the pods. These diagnostic tools enable Kubernetes to monitor the health of the workloads and take appropriate actions when issues arise, like restarting unhealthy containers or temporarily removing pods from service endpoints. Configuring probes is essential for maintaining the stability, resilience, and overall performance of applications in the Kubernetes cluster.
+
+## Overview of Probes
+
+Kubernetes supports three main types of probes, each serving a specific purpose:
+
+- **Liveness Probe:** Checks if a container is still running. If the liveness probe fails, Kubernetes restarts the container to recover from issues like deadlocks or crashes.
+- **Readiness Probe:** Determines if a container is ready to handle traffic. If the readiness probe fails, the Pod is removed from the Service's endpoint list, preventing traffic from being routed to it.
+- **Startup Probe:** Ensures that a container has successfully started. It is particularly useful for applications with long initialization times. Startup probes override liveness and readiness probes until they succeed.
+
+---
+
+## Probe Configuration
+
+Probes are configured as part of the Pod specification in YAML manifests. Each probe can define multiple parameters to suit specific application requirements. Below are the details for configuring each type of probe:
+
+#### Key Fields
+- **`path`**: The HTTP endpoint to query.
+- **`port`**: The container port to target.
+- **`initialDelaySeconds`**: Time to wait before starting the probe.
+- **`periodSeconds`**: Interval between probe attempts.
+- **`timeoutSeconds`**: Maximum time to wait for a response.
+- **`failureThreshold`**: Number of consecutive failures before the probe is considered failed.
+- **`successThreshold`**: Number of consecutive successes before the probe is considered successful.
```

Comment:
You don't need to highlight field names with BOTH teletype fonts and bold.
Simply make them teletype would be good enough.